common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -90,9 +90,10 @@ enum eGlobals {
     PHY_RATE_100KB_MAX                      = 8666,
     PHY_RATE_100KB_MIN                      = 72,
     PHY_RATE_100KB_INVALID                  = 0,
-    BSS_STEER_DISASSOC_TIMER_MS             = 200, // ~200ms
-    BSS_STEER_IMMINENT_VALID_INT            = 100,
-    BSS_STEER_VALID_INT                     = 50,
+    BEACON_TX_TIME_MS                       = 100,
+    BSS_STEER_DISASSOC_TIMER_MS             = 2000,
+    BSS_STEER_IMMINENT_VALID_INT            = (BSS_STEER_DISASSOC_TIMER_MS / BEACON_TX_TIME_MS),
+    BSS_STEER_VALID_INT                     = 2, // 200ms
```

Comment:
I think it will be a good idea to rename it and add the unit to the name:
BSS_STEER_VALID_BEACON_INTERVAL

Reply:
changed to BSS_STEER_VALID_INT_BTT